rfcs/variables-naming.md

```diff
@@ -0,0 +1,13 @@
+# Orbit Variables naming convention
```

Comment:
The name is too vague, Orbit have a lot of type of variables. JS variables, CSS variables, environment variables, ...

This RFC should

Reply:
One thing that would be really usefull is to understand the scope behind all this.

What goes in foundation? What goes in a theme?

For example, does `--o-ui-alias-primary-50` goes in foundation? It's not a token.